/*
 * This file was automatically generated by EvoSuite
 * Wed Apr 28 14:28:26 GMT 2021
 */

package com.ufcg.sad.exceptions;

import org.junit.Test;
import static org.junit.Assert.*;
import com.ufcg.sad.exceptions.EntidadeInvalidaException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class EntidadeInvalidaException_ESTest extends EntidadeInvalidaException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      EntidadeInvalidaException entidadeInvalidaException0 = new EntidadeInvalidaException();
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      EntidadeInvalidaException entidadeInvalidaException0 = new EntidadeInvalidaException("");
  }
}
